### PR TITLE
Bring back logic which is required for deep-copy auto-generation.

### PR DESCRIPTION
**Pull request by vikaschoudhary16**
Changes in this PR are required to generate `pkg/apis/awsproviderconfig/v1alpha1/zz_generated.deepcopy.go`

This was got removed in this commit, https://github.com/openshift/cluster-api-provider-aws/pull/129/commits/9ba17c501ef1253ba4297471b6a0561b160ab978

### CODE REVIEW

File: hack/boilerplate/boilerplate.go.txt [hunk 1]
```diff
@@ -0,0 +1,15 @@
+/*
```
Comment (enxebre) [inline hack/boilerplate/boilerplate.go.txt]: do we really need any other bolierplate than this one?

Comment (vikaschoudhary16) [reply hack/boilerplate/boilerplate.go.txt]: seems not. However i copied all from upstream just to have minimum differences wherever possible.

**Pull request merged**